subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/GradlePomModuleDescriptorParserTest.groovy

```diff
@@ -204,7 +204,7 @@ class GradlePomModuleDescriptorParserTest extends AbstractGradlePomModuleDescrip
 <project>
     <modelVersion>4.0.0</modelVersion>
     <groupId>group-one</groupId>
-    <artifactId>artifact-one</artifactId>
+    <artifactId>parent</artifactId>
```

Comment:
It was actually intentional, if you look at some lines above, this xml
points to a variable called parent.
The one below, called pomFile has the same artifactId, artifact-one.
I suspect the previous committer to have make a typo here.